lighthouse-cli/run.js

```diff
@@ -159,66 +163,62 @@ async function saveResults(runnerResult, flags) {
   }
 }
 
+/**
+ * Attempt to kill the launched Chrome, if defined.
+ * @param {ChromeLauncher.LaunchedChrome=} launchedChrome
+ * @return {Promise<void>}
+ */
+async function potentiallyKillChrome(launchedChrome) {
+  if (!launchedChrome) return;
+
+  try {
+    await launchedChrome.kill();
```

Comment:
should we time this out?

Reply:
> should we time this out?

probably, but the Chrome child process still keeps the main process going, so it just prints a `Couldn't quit Chrome process. Timed out.` error message but still sits there spinning. Is that better?

We could also `process.exit(1)` in that case, but assuming we want to always exit in that case seems like it might be excessive?